.github/CONTRIBUTING.md

```diff
@@ -1,39 +0,0 @@
-## How to contribute
-
-Thank you for your interest in contributing to Pattern Manager by WP Engine! There are many ways to contribute to this project.
```

Comment:
This is so well written, it's painful to remove it.

But now that we're not working on PM, it sends the wrong message.